app-services/secrets/res/configuration.toml

```diff
@@ -52,17 +51,35 @@ Host = "localhost"
 Port = 27017
 Timeout = "30s"
 
+# Shared Secret Store
 [SecretStore]
   Host = 'localhost'
   Port = 8200
-  Path = '/v1/secret/edgex/appservice'
+  Path = '/v1/secret/edgex/edgex-appservice/'
   Protocol = 'http'
-  Tokenfile = 'res/token.json'
+  TokenFile = './res/token.json'
```

Comment:
Missing `  RootCaCertPath = '/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem' `

Reply:
the compose file in developer scripts have `SecretStore_RootCaCertPath: /tmp/edgex/secrets/ca/ca.pem` in the common env variables

Reply:
That is for Docker. This is the value used in all the TOML files for non-docker.